FilmFestivalLoader/.idea/.gitignore

```diff
@@ -1,8 +0,0 @@
-# Default ignored files
-/shelf/
```

Comment:
Please restore (or explain why it is gone).

Reply:
Copied it to `~Stashed`.

Will store it as soon as the #286 brach is active again.